rasa/core/actions/action.py

```diff
@@ -625,7 +625,14 @@ async def _utter_responses(
         """Use the responses generated by the action endpoint and utter them."""
         bot_messages = []
         for response in responses:
+            generated_response = None
             generated_response = response.pop("response", None)
+            if not generated_response:
+                generated_response = response.pop("template", None)
+                rasa.shared.utils.io.raise_deprecation_warning(
+                    "The terminology 'template' is deprecated and replaced by 'response', use the `response` parameter instead of `template` in the dispatcher.",
```

Comment:
How about making this warning more actionable? E.g. upgrading to Rasa SDK xy or adapting their custom SDK?